app/src/controllers/filter/selectors.js

```diff
@@ -68,3 +74,19 @@ export const dirtyFilterIdsSelector = createSelector(
       })
       .map((item) => item.id),
 );
+
+export const getLaunchesPageLinkSelector = createSelector(
```

Comment:
May be better `getLaunchFilterLinkSelector`

Reply:
I suggest to rename it to `createLaunchesFilterLinkSelector`. May be it would be better to move it to launches controller?